spec.mediawiki

```diff
@@ -588,7 +605,7 @@ for a particular market proceeds as follows.
 
 # Client connects to server and both identify themselves using a long-lived PKI keypair after the associated client has paid a (one-time) fee.
 
-# Server responds with a fee that the client must pay to get a trading account on the server. Separeate fees may be required for individual markets since the potential for abuse may vary based on the market lot sizes or order book.
+# Server responds with a fee that the client must pay to get a trading account on the server. Separate fees may be required for individual markets since the potential for abuse may vary based on the market lot sizes or order book.
```

Comment:
As per discussion elsewhere, we will probably drop this. This whole section needs a rewrite though, because it doesn't define the messaging protocol yet.